community/bolt/src/docs/dev/messaging.asciidoc

```diff
@@ -76,7 +76,9 @@ This message is always the first message the client sends after <<bolt-handshake
 Sending any message other than `INIT` as the first message to the server will result in a `FAILURE` and the
 server closing the connection.
 
-The server will reply with a `SUCCESS` message unless some parameter for initialization is malformed.
+===== Response
```

Comment:
Consider replacing the new section titles with block titles, i.e., `.Response`. This would give a nice rhythm of `.Response` followed by `.Example`.
Is the following sentence beginning "All parameters in the `INIT`" part of the new section? It doesn't look like it is, in which case it should be moved up above the response block/section. This would also further enforce said rhythm.